### PR TITLE
[MQ-174] Beamer acceptance test

### PR DESCRIPTION
**Pull request by michelleshull**

### CODE REVIEW

File: tests/acceptance/BeamerAcceptanceCest.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+class BeamerAcceptanceCest {
+  function openAndCloseBeamer(\AcceptanceTester $I) {
+    $I->wantTo('Open and close Beamer sidebar');
+    $I->login();
+    $I->amOnMailPoetPage('Emails');
+    $I->click('.mailpoet_feature_announcement_icon');
+    $I->waitForElement('#beamerNews');
+    $I->switchToIframe('beamerNews');
+    $I->click('.headerClose');
+    $I->dontSeeElement('#beamerNews');
```
[the remark this answers is not in the document]
Comment (michelleshull) [reply tests/acceptance/BeamerAcceptanceCest.php]: Agreed

File: tests/acceptance/BeamerAcceptanceCest.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class BeamerAcceptanceCest {
+  function openAndCloseBeamer(\AcceptanceTester $I) {
+    $I->wantTo('Open and close Beamer sidebar');
+    $I->login();
+    $I->amOnMailPoetPage('Emails');
+    $I->click('.mailpoet_feature_announcement_icon');
+    $I->waitForElement('#beamerNews');
+    $I->switchToIframe('beamerNews');
+    $I->click('.headerClose');
```
[the remark this answers is not in the document]
Comment (michelleshull) [reply tests/acceptance/BeamerAcceptanceCest.php]: I disagree, and adding it made the test fail. My hypothesis is that the iframe still exists, with the elements we're asserting aren't visible inside, so if we check in the iframe, they're still going to exist, even if the user can't see them. (This seems supported by watching the test run in VNC in both this committed state, and with the additional switch to iframe, but I might be missing something.)

[the remark this answers is not in the document]
Comment (NeosinneR) [reply tests/acceptance/BeamerAcceptanceCest.php]: But looking at the previous assertions, we are testing that there is element `#beamerNews`, then switching inside it, so it only makes sense to switch back to the parent to see, that it's not there after closing it. Also checking that `#beamerSelector` is not visible also should be checked from the parent.

Just so we are on the same page, `$I->switchToIFrame();` is used to switch out of any iframe, back to parent.

[the remark this answers is not in the document]
Comment (michelleshull) [reply tests/acceptance/BeamerAcceptanceCest.php]: After a chat in Slack, Jan is totally right here. This commit should do it.

**Pull request merged**